.gitignore

```diff
@@ -0,0 +1,49 @@
+### OSX ###
+.DS_Store
+*/.DS_Store
+.DS_Store?
+.AppleDouble
+.LSOverride
+
+# Icon must end with two \r
+Icon
```

Comment:
This seems a weird one to me. Will we not land up with issues when someone wants to commit a directory 'Icon'? What generates this that we don't want to commit it?